packages/desktop/src/services/test/FileViewerServiceElectron.test.ts

```diff
@@ -69,4 +73,41 @@ describe(`${RUN_IN_RENDERER} FileViewerServiceElectron`, () => {
             expect(attemptedToShowError).to.be.true;
         });
     });
+
+    describe("openNativeFileBrowser", () => {
+        let existsSyncStub: SinonStub;
+
+        before(() => {
+            existsSyncStub = stub(fs, "existsSync");
```

Comment:
You can use the sandbox established before
```suggestion
            existsSyncStub = sandbox.stub(fs, "existsSync");
```

Reply:
Also could remove the `after` two lines below this

Reply:
Done in 8c20038b034e59c10dc4284be039a40b071a5ec1